pkg/health/health_node.go

```diff
@@ -12,12 +12,14 @@ import (
 func getNodeHealth(obj *unstructured.Unstructured) (*HealthStatus, error) {
 	var node v1.Node
 	if err := runtime.DefaultUnstructuredConverter.FromUnstructured(obj.Object, &node); err != nil {
-
 		return nil, fmt.Errorf("failed to convert unstructured Node to typed: %v", err)
 	}
+
 	for _, cond := range node.Status.Conditions {
 		if cond.Type == v1.NodeReady && cond.Status == v1.ConditionTrue {
 			return &HealthStatus{
+				Ready:  true,
```

Comment:
I think ready should default to true if not specified

Reply:
most of the cases result in `Ready` being false. So, I think this is fine.